subsys/net/lib/nrf_cloud/src/nrf_cloud_fota_poll.c

```diff
@@ -141,6 +142,11 @@ static void http_fota_dl_handler(const struct fota_download_evt *evt)
 		} else {
 			k_sem_give(&fota_download_sem);
 		}
+
+		if (ctx_ptr->status_fn) {
```

Comment:
will this be reached?
looks like it will go into the if statement above and reboot:
```
if (ctx_ptr->is_nonblocking) {
	handle_download_succeeded_and_reboot(ctx_ptr);
}
```

Reply:
Good catch, I think it only worked in the oob case because we don't call reboot directly in the callback.